.yamato/scripts/build_classlibs.bat

```diff
@@ -0,0 +1,20 @@
+@echo OFF
+
+if not exist "incomingbuilds" mkdir "incomingbuilds"
+
+for %%x IN ("windows", "OSX", "Linux") do (
+    ECHO build.cmd libs -os %%x -c release
+    build.cmd libs -os %%x -c release
+    if NOT %errorlevel% == 0 (
+        echo "build failed"
+        EXIT /B %errorlevel%
+    )
+    if not exist "incomingbuilds/coreclrjit-%%x" mkdir "incomingbuilds/coreclrjit-%%x"
+    ECHO xcopy /s /e /h /y "artifacts/bin/runtime/net7.0-%%x-Release-x64" "incomingbuilds/coreclrjit-%%x"
+    xcopy /s /e /h /y "artifacts/bin/runtime/net7.0-%%x-Release-x64" "incomingbuilds/coreclrjit-%%x"
+    ECHO taskkill /IM "dotnet.exe" /F
+    taskkill /IM "dotnet.exe" /F
+    echo build.cmd -clean
+    build.cmd -clean
```

Comment:
I've seen this hang once if the build fails... not sure if it was a fluke or there's something wrong in here.